src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,354 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import {
+  Box,
+  Button,
+  createStyles,
+  FormControl,
+  ListSubheader,
+  MenuItem,
+  Select,
+  TextField,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+
```

Comment:
```suggestion
```
Avoid newlines in the import section, unless there's a clear separation between different assets. (e.g images vs components)